src/App.js

```diff
@@ -1,21 +1,21 @@
 import './App.css';
 import React, { Component } from 'react'
 import Find from './Find'
-import CardFront from './CardFront'
+import favoritePlayer from './SinglePlayerData.json'
```

Comment:
This is so that I can show the card of the favorited player when the app loads. Right now it's a constant Javy Baez, but eventually the user will be able to set their fave.